### PR TITLE
Add setNftMetadata

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:
- add setNftMetadata
- get options when add nft symbol

[setNftMetadata result](https://console.firebase.google.com/u/0/project/nft-server-dev/database/nft-server-dev-default-rtdb/data/~2FnftMetadataHosting~2FETH-TEST~2Fgoerli~2F0xe18Daf5488f9E88fedE977452B5E0b1787cAf253~2F171~2Fmetadata)

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/asset.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import { AppCreditInfo, HttpMethod, NftContractBySymbol, NftToken, NftCollections } from './types';
+import { AppCreditInfo, HttpMethod, NftContractBySymbol, NftToken, NftCollections, NftMetadata } from './types';
 
 export default class Asset extends AinftBase {
   addNftSymbol(
     appId: string,
     chain: string,
-    contractAddress: string
+    contractAddress: string,
+    options?: Record<string, any>
```
Comment (comcom-jiyoung) [inline src/asset.ts:9]: What is the `options`?

**Review by comcom-jiyoung — APPROVED** [review]
LGTM, Thank you 👍
Then I can use this `setNftMetadata` API of ainft-js, right?

File: src/asset.ts (hunk 1 above)
Comment (ehgmsdk20) [reply src/asset.ts:9]: Yes, you can use `setNftMetadata` after adding your contract to nft-server using `addNftSymbol`.
If you want to use set metadata api, you should add nft symbol with option: `{ isDynamic: true }`.
This option will prevent nft-pipeline from updating metadata itself.

Comment (jiyouang) [reply src/asset.ts:9]: I got it, thank you for your kind explanation!

**Comment by ehgmsdk20** [comment]
Thanks for the reviews! Now merging~

**Pull request merged**